### PR TITLE
added quotes in english and dutch

### PR DESCRIPTION
**Pull request by J4spr**
### Checks

- [ ] Adding a language or a theme?
    - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
    - [ ] If is a theme, did you add the theme.css?
        - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [ ] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [ ] Make sure to include your GitHub username inside parentheses at the end of the PR title

Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/7388780646

File: frontend/static/quotes/dutch.json [hunk 1]
```diff
@@ -378,6 +378,12 @@
       "source": "Ton Lemaire",
       "length": 127,
       "id": 62
+    },
+    {
+      "text": "Zelf iets oplossen wil niet zeggen dat je het alleen moet doen.",
```
Comment (Miodec) [inline frontend/static/quotes/dutch.json:383]: Please provide a translation

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38712,6 +38712,60 @@
...
+      "text": "Love is something eternal: the aspects may change but not the essence.",
+      "source": "Vincent Van Gogh",
+      "lenght": 70,
+      "id": 7028
+    },
+    {
+      "text": "When I saw you I  fell in love, and you smiled because you knew.",
+      "source": "William Shakespeare",
+      "lenght": 64,
+      "id": 7029
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38768]: All of them have misspelled "length". Also most of them are too short (60 characters minimum please). Also some are missing a punctuation at the end.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/dutch.json (hunk 1 above)
Comment (NipulM) [reply frontend/static/quotes/dutch.json:383]: It translates to "Solving something by yourself doesn't mean you have to do it alone." in English.

**Pull request closed without merging**